src/doc/trpl/documentation.md

```diff
@@ -294,3 +294,20 @@ Documentation` on the first line).
 Like with a Rust crate, the `--test` argument will run the code
 examples to check they compile, and obeys any `--test-args` flags. The
 tests are named after the last `#` heading.
+
+# Re-exports
+
+Rustdoc will show the documentation for a publc re-export in both places:
+
+```{rust,ignore}
+extern crate foo;
+
+pub use foo::bar;
+```
+
+This will create documentation for `bar` both inside the documentation for
+the crate `foo`, as well as the documentation for your crate. It will use
+the same documentation in both places.
+
+In the standard library, this leads to much documentation that's shown in
+both `std` and `core`, since `std` re-exports a lot of `core`.
```

Comment:
While true, this doesn't necessarily seem like it belongs in the docs. I would expect this to be more of a bug report then anything else.